Loumir update preview (#51)
ivoa/HighEnergyObsCoreExt: HighEnergyObsCoreExt.tex

\ivoagroup{High Energy Interest Group}

\author{
	I. Evans (SAO/CXC, \href{mailto:ievans@cfa.harvard.edu}{ievans@cfa.harvard.edu},\\
	M. Servillat (LUX - Observatoire de Paris, \href{mailto:mathieu.servillat@obspm.fr}{mathieu.servillat@obspm.fr},\\

> 	B. Khélifi (APC - Université de Paris Cité/CNRS, \href{mailto:khelifi@in2p3.fr}{khelifi@in2p3.fr}),\\

Reviewer: ```suggestion
	B. Kh\'elifi (APC - Universit\'e de Paris Cit\'e/CNRS, \href{mailto:khelifi@in2p3.fr}{khelifi@in2p3.fr}),\\
```

	J. Evans (SAO/CXC, \href{mailto:janet@cfa.harvard.edu}{janet@cfa.harvard.edu}),\\

> 	M. Louys (CDS and ICube - Université de Strasbourg, \href{mailto:mireille.louys@unistra.fr}{mireille.louys@unistra.fr}),\\

Reviewer: ```suggestion
	M. Louys (CDS and ICube - Universit\'e de Strasbourg, \href{mailto:mireille.louys@unistra.fr}{mireille.louys@unistra.fr}),\\
```

	M. Kettenis (Joint Institute for VLBI ERIC, \href{mailto:kettenis@jive.eu}{kettenis@jive.eu}),\\
	F. Bonnarel (IVOA, \href{mailto:francois.bonnarel@gmail.com}{francois.bonnarel@gmail.com}),\\
	L. Michel (SSC-XMM/SVOM - Strasbourg Observatory, \href{mailto:laurent.michel@astro.unistra.fr}{laurent.michel@astro.unistra.fr}),\\
	C. Boisson (LUX - Observatoire de Paris, \href{mailto:catherine.boisson@obspm.fr}{catherine.boisson@obspm.fr}),\\
	M. Cresitello-Dittmar (SAO/CXC, \href{mailto:mdittmar@cfa.harvard.edu}{mdittmar@cfa.harvard.edu}),\\
	O. Ates (LUX - ObsParis, \href{mailto:onur.ates@obspm.fr}{onur.ates@obspm.fr}),\\

> 	K. Kosack (IRFU - CEA/Université Paris-Saclay, \href{mailto:karl.kosack@cea.fr}{karl.kosack@cea.fr}),\\

Reviewer: ```suggestion
	K. Kosack (IRFU - CEA/Universit\'e Paris-Saclay, \href{mailto:karl.kosack@cea.fr}{karl.kosack@cea.fr}),\\
```

\section{Proposed ivoa.obscore\_hea Table Attributes}\label{sec:ibscoreext}

This section summarizes the proposal for the \gls{HEA} extension of ObsCore.  We use the term {\em ivoa.obscore\_hea\/} to described the extension here and in Appendix~\ref{sec:uc}.
% start mireille ucd update

> \begin{longtable}{ | m{2.5cm} | m{4em} | m{3em} | m{3em} | m{6cm} | m{2.3em} |}

Reviewer 2: This does not fit a page width. To be changed

\hline

{\centering \bf Column Name}     &{\centering \bf UCD} &{\centering \bf Unit} &{\centering \bf Type} &{\centering \bf Description} &{\centering \bf MAN}\\
\hline
 ev\_xel & \ucd{meta.number;obs.event} &  unitless & int & {\footnotesize Number of events in an event\_list }& NO \\
\hline

>  s\_ref\_energy &  \ucd{meta.ref;em.energy;pos} & eV & float & {\footnotesize Energy at which the ObsCore spatial characterisation attributes s\_fov , s\_region, s\_resolution are defined} & NO \\

Reviewer 2: ```suggestion
 s\_ref\_energy &  \ucd{meta.ref;phys.energy;pos} & eV & float & {\footnotesize Energy at which the ObsCore spatial characterisation attributes s\_fov , s\_region, s\_resolution are defined} & NO \\
```

Reviewer: I think em.energy is correct here.

Author: so do I.

\hline

>  em\_ref\_energy  & \ucd{meta.ref;em.energy;em} & eV & float & {\footnotesize Energy at which the ObsCore spatial characterisation attributes em\_res\_power, em\_resolution are defined} & NO \\

Reviewer 2: ```suggestion
 em\_ref\_energy  & \ucd{meta.ref;phys.energy} & eV & float & {\footnotesize Energy at which the ObsCore spectral characterisation attributes em\_res\_power, em\_resolution are defined} & NO \\
```

Reviewer: Id.

\hline
 s\_ref\_oaa & \ucd{pos.posAng;instr.offset;pos} & deg & float & {\footnotesize Off-axis angle (i.e., the angular separation of the target or source from the telescope optical axis) at which the ObsCore spatial characterisation attributes s\_fov , s\_region, s\_resolution are defined} & NO \\
\hline

>  em\_ref\_oaa & \ucd{pos.posAng;instr.offset;em} & deg & float & {\footnotesize Off-axis angle (i.e., the angular separation of the target or source from the telescope optical axis) at which the ObsCore spectral characterisation attributes em\_res\_power, em\_resolution are defined} & NO \\

Reviewer 2: "pos.posAng;instr.offset;em" or "pos.posAng;instr.offset", because events are not necessary photons...

\hline
  t\_intervals & \ucd{?? }& unitless & TMOC & {\footnotesize List of observation intervals or stable/good time intervals describing the exact observation time coverage} & NO \\
\hline

>   energy\_min & \ucd{em.energy;stat.min} & float & eV &  {\footnotesize Energy associated to the Obscore attribute em\_max, describing the minimal energy of the dataset} & NO \\

Reviewer 2: ```suggestion
  energy\_min & \ucd{phys.energy;stat.min} & float & eV &  {\footnotesize Energy associated to the Obscore attribute em\_max, describing the minimal energy of the dataset} & NO \\
```

Reviewer: Id.
